docs/source/en/tasks/image_classification.mdx

```diff
@@ -130,17 +136,57 @@ To apply the preprocessing function over the entire dataset, use 🤗 Datasets [
 >>> food = food.with_transform(transforms)
 ```
 
-Now create a batch of examples using [`DataCollatorWithPadding`]. Unlike other data collators in 🤗 Transformers, the `DefaultDataCollator` does not apply additional preprocessing such as padding.
+
+Now create a batch of examples using [`DefaultDataCollator`]. Unlike other data collators in 🤗 Transformers, the `DefaultDataCollator` does not apply additional preprocessing such as padding.
 
 ```py
 >>> from transformers import DefaultDataCollator
 
 >>> data_collator = DefaultDataCollator()
 ```
+</pt>
+</frameworkcontent>
+
+
+<frameworkcontent>
+<tf>
+Create a preprocessing function to resize the images and return the `pixel_values`, which are required inputs to the model.
+
+```py
+>>> def process(examples):
+...     examples.update(
+...         image_processor(
+...             examples["image"],
+...         )
+...     )
```

Comment:
I had it in one line initially but `make style` converted it this way. If I revert it back it'll likely fail on the CI side.

Reply:
I think `make style` made it across several lines because `black` will [split across many lines if there's a comma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma) and no element after it e.g. 

```python
foo = bar(a, b, c,)
```

will be set to 

```python
foo = bar(
    a,
    b,
    c,
)
```

In the suggestion I posted above it should allow you to leave it on one line.